src/signalingclient/DefaultSignalingClient.ts

```diff
@@ -40,6 +40,7 @@ export default class DefaultSignalingClient implements SignalingClient {
   private isClosing: boolean;
   private connectionRequestQueue: SignalingClientConnectionRequest[];
   private unloadHandler: () => void | null = null;
+  private audioSessionId: number = Math.trunc(Math.random() * Number.MAX_SAFE_INTEGER);
```

Comment:
`audioSessionId` is defined as a `uint64` but this math only uses the lowest 53-bits of the session id (up to MAX_SAFE_INTEGER), thus the `uint64` values will not be uniformly distributed. I would suggest composing the `uint64` as a [Long](https://www.npmjs.com/package/long) in terms of two 32-bit integers (low and high word). Also, I suggest using [Crypto.getRandomValues()](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/getRandomValues) to ensure the randomness is seeded with entropy. Also break this out into a separate function and write a smoke test for the randomness (e.g. check that N=1000 random ids are unique, check that average of N ids is between `0x7000000000000000` and `0x9000000000000000`, i.e. near the midpoint of the 64-bit range).

Reply:
Why is 53 bits not enough for our purposes? It would still be extremely unlikely session ids would be the same.

Reply:
I see how it makes sense from a security standpoint though. I'll make the change.

Reply:
It's more about honoring the existing server-client contract than whether it is unique enough. The server expects the client to generate a uniformly distributed 64-bit integer, so we should try to maintain that.